cmd/cni-metrics-helper/README.md

```diff
@@ -15,24 +15,29 @@ The following diagram shows how `cni-metrics-helper` works in a cluster:
 As you can see in the diagram, the `cni-metrics-helper` connects to the API Server over https (`tcp/443`), and another connection is created from the API Server to the worker node over http (`tcp/61678`). If you deploy Amazon EKS with the recommended security groups from [Restricting cluster traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html#security-group-restricting-cluster-traffic), then make sure that a security group is in place that allows the inbound connection from the API Server to the worker nodes over `tcp/61678`.
 
 Adding the CNI metrics helper will publish the following metrics to CloudWatch:
-```
-"addReqCount",
-"assignIPAddresses",
-"awsAPIErr",
-"awsAPILatency",
-"awsUtilErr",
-"delReqCount",
-"eniAllocated",
-"eniMaxAvailable",
-"ipamdActionInProgress",
-"ipamdErr",
-"maxIPAddresses",
-"podENIErr",
-"reconcileCount",
-"totalIPAddresses",
-"totalIPv4Prefixes",
-"totalAssignedIPv4sPerCidr"
-```
+
+| Metric | Description |
+| ------ | ----------- |
+| addReqCount | Aggregate of the number of CNI ADD requests that require an IP address |
+| assignIPAddresses | Aggregate of the number of IP addresses assigned to pods |
+| awsAPIErr | Aggregate of the number of times AWS API returns an error |
+| awsAPILatency | The maximum of AWS API call latency in ms |
+| awsUtilErr | Aggregate of the number of errors not handled in awsutils library |
+| delReqCount | Aggregate of the number of delete IP address requests |
+| eniAllocated | Aggregate of the number of ENIs allocated |
+| eniMaxAvailable | Aggregate of the maximum number of ENIs that can be attached to this instance, accounting for unmanaged ENIs |
+| ipamdActionInProgress | Aggregate of the number of ipamd actions in progress |
+| ipamdErr | Aggregate of the number of errors encountered in ipamd |
+| maxIPAddresses | Aggregate of the maximum number of IP addresses that can be allocated to the instance |
+| podENIErr | Aggregate of the number of errors encountere for pod ENIs |
```

Comment:
This can be reworded as: "The number of errors encountered while managing ENIs for pods, summed across all nodes."